node.config.test.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://nodejs.org/dist/v24.10.0/docs/node-config-schema.json",
```

Comment:
in nvmrc we use 25 we should use a link like `/v25-x/docs/node-config-schema.json`

Reply:
It's kinda complicated / annoying because we test against multiple versions of node, and the config file has varying support across versions :/

Reply:
oh ...